Core/src/org/sleuthkit/autopsy/casemodule/StartupWindowProvider.java

```diff
@@ -84,35 +88,61 @@ private void init() {
                 checkSolr();
             }
 
+            File openPreviousCaseFile = new File(ResetWindowsAction.getCaseToReopenFilePath());
+            String caseFilePath = "";
+            if (openPreviousCaseFile.exists()) {
+                try {
+                    Charset encoding = null;
+                    caseFilePath = FileUtils.readFileToString(openPreviousCaseFile, encoding);
+                    if (new File(caseFilePath).exists()) {
+                        FileUtils.forceDelete(openPreviousCaseFile);
+                        Case.openAsCurrentCase(caseFilePath);
+                        //the case is now open we do not want to display the start up windows
+                        return;
+                    } else {
+                        logger.log(Level.WARNING, "Unable to open previously open case because metadata file not found at: {0}", caseFilePath);
+                    }
+                } catch (IOException ex) {
+                    logger.log(Level.WARNING, "Unable to open or delete file containing path " + ResetWindowsAction.getCaseToReopenFilePath() + " to previously open case, will not open previous case.", ex);
+                } catch (CaseActionException ex) {
+                    logger.log(Level.WARNING, "Unable to open previously open case with metadata file: " + caseFilePath, ex);
```

Comment:
I would think a popup instead of logging would be viable here.

Reply:
A pop up should be viable here, it just didn't seem helpful, it tells them something they have 0 recourse about, and creates another pop up while Autopsy is trying to open and initialize all its current pop ups and user interface, all as a result of 'something unexpected happened, and so we instead of keep you stuck in a loop of always opening the same case every time you open autopsy we have opted to not open any case'